### PR TITLE
msdkenc: delelte internal vpp

### PR DESCRIPTION
**Pull request by liuyinhangx**

### CODE REVIEW

File: subprojects/gst-plugins-bad/sys/msdk/gstmsdkenc.c [hunk 1]
```diff
@@ -82,15 +82,13 @@ GST_DEBUG_CATEGORY_EXTERN (gst_msdkenc_debug);
 static GstStaticPadTemplate sink_factory = GST_STATIC_PAD_TEMPLATE ("sink",
     GST_PAD_SINK,
     GST_PAD_ALWAYS,
-    GST_STATIC_CAPS (GST_MSDK_CAPS_STR
-        ("{ NV12, I420, YV12, YUY2, UYVY, BGRA }", "NV12"))
```
Comment (mengker33) [inline subprojects/gst-plugins-bad/sys/msdk/gstmsdkenc.c:86]: According to media-driver https://github.com/intel/media-driver#decodingencoding-features, 8bit/422 is supported for hevc encoder from platforms > ICL, so we may include yuy2 in the caps and add a platform check somewhere else (like in gst_msdkenc_init_encoder ? ).

File: subprojects/gst-plugins-bad/sys/msdk/gstmsdkh265enc.c [hunk 2]
```diff
@@ -101,7 +101,7 @@ enum
 #define PROP_INTRA_REFRESH_CYCLE_DIST_DEFAULT 0
 #define PROP_DBLK_IDC_DEFAULT                 0
 
-#define RAW_FORMATS "NV12, I420, YV12, YUY2, UYVY, BGRA, BGR10A2_LE, P010_10LE, VUYA"
```
Comment (mengker33) [inline subprojects/gst-plugins-bad/sys/msdk/gstmsdkh265enc.c:104]: According to media-driver https://github.com/intel/media-driver#decodingencoding-features, 8bit/422 is supported for hevc encoder from platforms > ICL, so we may include yuy2 in the caps and add a platform check somewhere else (like in gst_msdkenc_init_encoder ? ).

Comment (liuyinhangx) [reply subprojects/gst-plugins-bad/sys/msdk/gstmsdkh265enc.c:104]: Thank you, @mengker33 , I will make further confirmation of all the formats.

**Comment by uartie** [comment]
No activity for a while. Closing to free CI resources.

**Pull request closed without merging**